src/Security/SecurityConfigUpdater.php

```diff
@@ -92,6 +92,21 @@ public function updateForAuthenticator(string $yamlSource, string $firewallName,
         }
 
         $newData['security']['firewalls'][$firewallName] = $firewall;
+
+        $accessControlRules = $newData['security']['access_control'] ?? [];
+
+        foreach ($accessControlRules as $rule) {
+            if ('^/login$' === $rule['path']) {
```

Comment:
I think that check is a little bit too strict: I think we should check for `^/login`, it's the syntax from the (old) documentation: https://symfony.com/doc/4.0/security/form_login_setup.html#be-sure-the-login-page-isn-t-secure-redirect-loop

Reply:
Fixed!